core/src/main/java/io/grpc/ForwardingChannelBuilder.java

```diff
@@ -91,9 +92,20 @@ public T overrideAuthority(String authority) {
     return thisT();
   }
 
+  /**
+   * @deprecated use {@link #usePlaintext()} instead.
+   */
   @Override
+  @Deprecated
   public T usePlaintext(boolean skipNegotiation) {
-    delegate().usePlaintext(skipNegotiation);
+    @SuppressWarnings("deprecation")
```

Comment:
Is this necessary? I thought `@Deprecated` methods could call other `@Deprecated` methods.

Reply:
Done.